### PR TITLE
Temporary PR for reviewing initial commit history.

### PR DESCRIPTION
**Pull request by colinhoglund**
This is all already on the `master` branch, but using temporary branches so that the POC code can be reviewed.

This PR will not be merged and any necessary changes will be made in a followup commit.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,5 @@
+module github.com/colinhoglund/drone-bazelisk-ecr
```
Comment (colinhoglund) [inline go.mod:1]: I will eventually change this to `github.com/mongodb-forks/drone-bazelisk-ecr`

Comment (dskatz) [reply go.mod:1]: 👍

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ENV ECR_LOGIN_PATH /usr/local/bin/docker-credential-ecr-login
+
+RUN groupadd -g ${BAZEL_USER_ID} -r ${BAZEL_USER} \
+ && useradd -lmr -u ${BAZEL_USER_ID} -g ${BAZEL_USER} ${BAZEL_USER} \
+ && apt-get update && apt-get install -y \
+      g++ \
+      git \
+      unzip \
+      wget \
+      zip \
```
Comment (dskatz) [inline Dockerfile:28]: Are these bazel runtime dependencies?
g++, git, unzip, zip?

File: cmd/drone-bazelisk-ecr/plugin.go [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+)
+
+// plugin configuraion
+type plugin struct {
+	Target     string `required:"true"`
+	Registry   string `required:"true"`
+	Repository string
+	Tag        string
+	AccessKey  string `split_words:"true"`
+	SecretKey  string `split_words:"true"`
```
Comment (dskatz) [inline cmd/drone-bazelisk-ecr/plugin.go:18]: These are not required?

Comment (colinhoglund) [reply cmd/drone-bazelisk-ecr/plugin.go:18]: Technically no. For example, you could pass in `AWS_` env vars directly in the drone pipeline if you desired.

Comment (dskatz) [reply cmd/drone-bazelisk-ecr/plugin.go:18]: 👍

**Comment by colinhoglund** [comment]
Closing, thanks for review @dskatz.

**Pull request closed without merging**